common/protocol/Ola.proto

```diff
@@ -266,6 +266,8 @@ message UniverseInfo {
   required int32 input_port_count = 4;
   required int32 output_port_count = 5;
   required int32 rdm_devices = 6;
+  repeated PortInfo input_ports = 7;
```

Comment:
You can't have a field that is optional AND repeated. We would have to create a new message type that contains the repeated fields, and add a new optional field of this new type. It seems like unnecessary complexity that we will have to keep in the long term.

As @nomis52 said, a missing field will appear to have the same length as a present field with 0 elements. But since we still have fields with the counts, you can still differentiate between the two situations (if input_port_count != len(inputs_ports), it means you are dealing with an old OLA that doesn't fill input_ports). So, it is really necessary to change the protobuf definition?